src/index.html

```diff
@@ -6,9 +6,55 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link
+         rel="preconnect"
+         href="https://fonts.googleapis.com"
+    >
+    <link
+         rel="preconnect"
+         href="https://fonts.gstatic.com"
+         crossorigin="anonymous"
+    >
+    <link
+         href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+         rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logoLink" href="index.html">
+        <img class="logo" src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1702252800&Signature=SaF7YW0H7Sok3Ym~DTrBU7ICffnIqRz85sOzyK5yhUxkE3UQ7SZVMqgwJzD7mdtcXPnACi7Ud-B6xHi8GNoLdzYQcKpuN4FSzrfnAW-HFEtzHrr8llePYN-4UVeqlsxGrWvAs4EmLpOGsCinSSqxoEUfy-jmuU3cRmGQH31GOeuBGQbY33Kv1udxQyCOd00xIqkEwSSPhItWv-Ihi0IHbMzH61-wKa1DabtMzHARFKwpbi-ULMUdDjVRxEl4E32z92yZqtRoguXwEVaTgr4GNGJ5UKwjJSJ5NNv3Qv0mwjGoFRAq67O7YmIO28GekphRU5W6kzgVJhd4HUe1Ul3zLg__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4"
+        alt="MOYO logo">
+      </a>
+      <nav class="nav">
+        <ul class="linkList">
+          <li>
+            <a id="firstLink" class="is-active navLink">Apple</a>
+          </li>
```

Comment:
Need to fix it everywhere
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="is-active nav__link">Apple</a>
          </li>
```